packages/ui-library/src/components/form-label/index.test.ts

```diff
@@ -102,12 +102,12 @@ describe('blr-form-label', () => {
     const element = await fixture(
       WrappedBlrFormLabelRenderFunction({
         ...sampleParams,
-        variant: 'error',
+        hasError: true,
       })
     );
     const blrLabel = querySelectorDeep('label.blr-form-label', element.getRootNode() as HTMLElement);
     const errorLabel = blrLabel?.getAttribute('class');
-    expect(errorLabel).to.contain('error');
+    expect(errorLabel).to.contain(true);
```

Comment:
May be it might be more reasonable to check for string [..to.contain('true')] rather than boolean OR if checking for boolean something like isTrue(...) might fit.

Reply:
Agreed. Changes done